src/main/java/net/minecraftforge/client/ForgeHooksClient.java

```diff
@@ -352,7 +352,9 @@ private LazyInit() {}
         return LazyInit.INSTANCE;
     }
 
+    // NO-OP method, kept for bin-compat. Used in TitleScreen.java
     public static void renderMainMenu(TitleScreen gui, GuiGraphics graphics, Font font, int width, int height, int alpha) {
```

Comment:
Techncially this entire class is intenal so we dont need to keep binary compatibility.
But also if you want to keep it do a deprecate forRemoval